src/Silk.Core/Tools/EventHelpers/IConfigService.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Silk.Core.Tools.EventHelpers
+{
+    internal interface IConfigService
```

Comment:
What's this doing? Is this what's being injected into the services? There should be IConfigService in Core/Services/Interfaces

Reply:
Correction: There isn't.